src/main/java/com/evilcorp/bloom/model/Category.java

```diff
@@ -0,0 +1,39 @@
+package com.evilcorp.bloom.model;
+
+import java.util.UUID;
+
+import org.springframework.data.annotation.Id;
+
+public class Category {
+
+  @Id
+  private UUID id;
```

Comment:
*general advice*: Keep in mind when it is better to use uuid vs sequencial id, and have a good answer for when to use it

Reply:
changed the id to be SERIAL/Integer upon discussion